modules/tinymce/CHANGELOG.md

```diff
@@ -6,66 +6,66 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- Triple-clicking did not select the paragraph in Google Chrome in some circumstances. #TINY-8215
+- Images were not showing as selected when selected along with other content. #TINY-5947
+- Selection direction was not stored or restored when getting or setting selection bookmarks. #TINY-8599
+- When text within an inline boundary element was selected and the right-arrow key was pressed, the insertion point incorrectly moved to the left. #TINY-8601
+- In some versions of Safari, the `editor.selection.isForward()` API could throw an exception due to an invalid selection. #TINY-8686
+- The selection is no longer incorrectly moved inside a comment by the `editor.selection.normalize()` API. #TINY-7817
+- The `InsertParagraph` or `mceInsertNewLine` commands did not delete the current selection like the native command does. #TINY-8606
+- The `InsertLineBreak` command did not replace selected content. #TINY-8458
+- If selected content straddled a parent and nested list, cutting the selection did not always set the list style to `'none'` on the parent list. #TINY-8078
+- Delete operations could behave incorrectly if the selection contains a `contenteditable="false"` element located at the edge of content. #TINY-8729
+- Spaces were not added correctly on some browsers when the insertion point was immediately before or after a `contenteditable` block element. #TINY-8588
+- Images that used a Data URI were corrupted when the data wasn't base64 encoded. #TINY-8337
+- `uploadImages` no longer triggers two change events if there is a removal of images on upload. #TINY-8641
+- Preview and Insert Template dialogs now display the correct content background color when using dark skins. #TINY-8534
+- Dialogs no longer exceed window height on smaller screens. #TINY-8146
+- UI components, such as dialogs, would in some cases cause the Esc keyup event to incorrectly trigger inside the editor. #TINY-7005
+- Fixed incorrect word breaks in menus when the menu presented with a scrollbar. #TINY-8572
+- Notifications did not properly reposition when toggling fullscreen mode. #TINY-8701
+- Text alignments, such as flush left and centered, could not be applied to `<pre>` elements. #TINY-7715
+- Indenting or outdenting list items inside a block element that was inside another list item did not work. #TINY-7209
+- Changing the list type of a list within another block element altered the parent element that contained that list. #TINY-8068
+- Pasting columns in tables could, in some circumstances, result in an invalid table. #TINY-8040
+- Copying columns in tables could sometimes result in an invalid copy. #TINY-8040
+- Custom elements added to otherwise blank lines were removed during serialization. #TINY-4784
+- Autocompleter was not triggered at the start of empty nested list items within non-empty list items. #TINY-8759
```

Comment:
```suggestion
- The editor's autocompleter was not triggered at the start of nested list items. #TINY-8759
```
This doesn't make any sense to me so just trying to make it, make sense... it's largely going back to the original. Also starting with `Autocompleter` doesn't make sense either. FYI @brian-forte so you can reword as needed.